### PR TITLE
Flight-Vision pipeline fixes

### PR DESCRIPTION
**Pull request by Ooglely**

### CODE REVIEW

**Comment by Ooglely** [comment]
Waiting for other branch to be merged to fix pre-commit issues

File: flight/camera.py [hunk 1]
```diff
@@ -1,16 +1,21 @@
 """A class that contains all the needed camera functionality for the drone."""
 
+# pylint: disable=too-many-locals
+# Not worth to deal with this with the time crunch we are in
```
Comment (cjhr95) [inline flight/camera.py:4]: bruh

File: flight/camera.py [hunk 2]
```diff
@@ -159,31 +158,35 @@ async def odlc_move_to(
...
-                        round(drone_long, int(6 * fast_param))
-                        == round(longitude, int(6 * fast_param))
-                    )
-                    and (round(drone_alt, 1) == round(altitude, 1))
-                ):
+                total_distance: float = calculate_distance(
+                    drone_lat, drone_long, drone_alt, latitude, longitude, altitude
+                )
+
+                if total_distance < WAYPOINT_TOLERANCE:  # 6 meters = 19.685 feet.
```
Comment (cjhr95) [inline flight/camera.py]: WAYPOINT_TOLERANCE is currently 1 meter, yet the comment indicates that its 6 meters. I would prefer a higher tolerance, so we can fly through the waypoints.

File: integration_tests/emg_obj_vision.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 """Contains the emergent object integration test pipeline."""
 
-from ctypes import c_bool
-from multiprocessing.sharedctypes import SynchronizedBase
+from ctypes import c_bool  # pylint: disable=unused-import
+from multiprocessing.sharedctypes import SynchronizedBase  # pylint: disable=unused-import
```
Comment (cjhr95) [inline integration_tests/emg_obj_vision.py:5]: If the imports aren't used anymore, delete them

File: state_machine/states/impl/land_impl.py [hunk 4]
```diff
@@ -41,7 +41,6 @@ async def run(self: Land) -> None:
                 break
             await asyncio.sleep(1)
 
-        await self.drone.system.action.disarm()
```
Comment (cjhr95) [inline state_machine/states/impl/land_impl.py:44]: Still might want this in some sort of error handling block

**Review by eblake003 — APPROVED** [review]
Vision stuff looks good to me

**Pull request merged**